packages/backend/src/plugins/search.ts

```diff
@@ -30,8 +30,7 @@ export default async function createPlugin({
   const indexBuilder = new IndexBuilder({ logger, searchEngine });
 
   indexBuilder.addCollator({
-    type: 'software-catalog',
-    defaultRefreshIntervalSeconds: 600,
+    defaultRefreshIntervalSeconds: 10,
```

Comment:
was this change supposed to be here? or was it to refresh more often when developing locally?

Reply:
It was to refresh more often when developing, right @iamEAP?

Reply:
Reset to 600 ms